src/issuer/issuer.controller.ts

```diff
@@ -19,38 +20,4 @@ export class IssuerController {
     public async createCredDef(@Body() body: any): Promise<any> {
         return await this.issuerService.createCredDef(body.schema_id, body.tag, body.support_revocation);
     }
-
```

Comment:
are these changes (and those below) going to break existing uses?

Reply:
I checked and we weren't calling them anywhere so figured there was no point keeping them around